REST/PowerShell/Reports/ListTenantedUnhealthyMachines.ps1

```diff
@@ -0,0 +1,86 @@
+$ErrorActionPreference = "Stop";
```

Comment:
Updating this to support TLS 1.2 and 1.3 (https://github.com/OctopusDeploy/OctopusDeploy-Api/blob/master/REST/PowerShell/Administration/GetUsageCounts.ps1#L328C1-L330C8) and adding a check that the variables were actually replaced (https://github.com/OctopusDeploy/OctopusDeploy-Api/blob/master/REST/PowerShell/Administration/GetUsageCounts.ps1#L9) fixes some issues I have seen in the past.